### PR TITLE
fix: resolve mobile screen search and repo link issues

### PR DESCRIPTION
**Pull request by lorypelli**
list of bugs I fixed
- Search on md screens
- Repo link that was going outside the container (reduced letter amount)
I did also some improvements to others file (if you don't like something, say it and I will change it)

### CODE REVIEW

File: .gitignore
Comment (vclemenzi) [inline .gitignore:1]: Because you cleaned the `.gitignore`

File: .dockerignore [hunk 1]
```diff
@@ -1,3 +1,2 @@
-.DS_Store
```
Comment (clemenzi) [inline .dockerignore]: Needed if someone starts docker from macOS

File: .gitignore [hunk 2]
```diff
@@ -1,21 +1,3 @@
...
-pnpm-debug.log*
-
-
-# environment variables
-.env
-.env.production
-
-# macOS-specific files
-.DS_Store
+node_modules
```
Comment (clemenzi) [inline .gitignore]: Why you cleaned the `.gitignore`?

**Review by clemenzi — APPROVED** [review]
LGTM!

**Pull request merged**